nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -679,13 +673,13 @@ private void addDocsServlets(WebAppContext webAppContext) {
         try {
             final File docsDir = getDocsDir();
 
-            final ServletHolder docs = new ServletHolder("docs", ResourceServlet.class);
+            final ServletHolder docs = new ServletHolder("docs", DefaultServlet.class);
             final Path htmlBaseResource = docsDir.toPath().resolve("html");
             docs.setInitParameter("baseResource", htmlBaseResource.toString());
             docs.setInitParameter("dirAllowed", "false");
             webAppContext.addServlet(docs, "/html/*");
 
-            final ServletHolder restApi = new ServletHolder("rest-api", ResourceServlet.class);
+            final ServletHolder restApi = new ServletHolder("rest-api", DefaultServlet.class);
```

Comment:
Is there a particular reason for changing from the ResourceServlet to the DefaultServlet? This previously resulted in warnings related to the DefaultServlet and path configuration.